src/overview/using-the-standard-library.md

```diff
@@ -67,55 +34,10 @@ graph TD;
     esp-idf-svc --> esp-idf-sys & esp-idf-hal & embedded-svc
 ```
 
+[embedded-svc]: https://github.com/esp-rs/embedded-svc
```

Comment:
```suggestion
### When you might want to use the Standard Library (`std`)

- Rich functionality: If your embedded system requires lots of functionality like support for networking protocols, file I/O, or complex data structures, you will likely want to use hosted-environment approach because std libraries provide a wide range of functionality that can be used to build complex applications relatively quickly and efficiently
- Portability: The `std` crate provides a standardized set of APIs that can be used across different platforms and architectures, making it easier to write code that is portable and reusable.
- Rapid development: The `std` crate provides a rich set of functionality that can be used to build applications quickly and efficiently, without worrying about low-level details.

[embedded-svc]: https://github.com/esp-rs/embedded-svc
```


I think adding this section from @JurajSadel [blogpost](https://blog.espressif.com/rust-embedded-a-development-power-duo-db43dae21206) can be really useful

Reply:
Definitely something to take into account, we could maybe add some clarifications.